core/src/main/java/fr/sncf/osrd/cli/WorkerCommand.kt

```diff
@@ -44,10 +47,12 @@ class WorkerCommand : CliCommand {
     val WORKER_POOL: String
     val WORKER_REQUESTS_QUEUE: String
     val WORKER_ACTIVITY_EXCHANGE: String
+    val ALL_INFRA: Boolean
 
     init {
-        WORKER_ID = System.getenv("WORKER_ID")
-        WORKER_KEY = System.getenv("WORKER_KEY")
+        ALL_INFRA = !System.getenv("ALL_INFRA").isNullOrEmpty()
+        WORKER_ID = if (ALL_INFRA) "all_infra_worker" else System.getenv("WORKER_ID")
```

Comment:
pretty sure this will conflict, could you rebase?

Reply:
It did, ended up refactoring a bit through a `getBooleanEnvvar` fun https://github.com/OpenRailAssociation/osrd/pull/9166/commits/52ffad3e68fc252d8e036c9ec76ba718383749ad#diff-7f5b2e605ea8cd19fd586f8d299a533f45c5419298dc3b16aca328645dee196aR75